config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -246,36 +246,6 @@ periodics:
   annotations:
     testgrid-dashboards: sig-node-containerd, sig-node-cri
     testgrid-tab-name: containerd-node-conformance
-- name: ci-containerd-node-e2e-1-2
-  interval: 1h
-  labels:
-    preset-service-account: "true"
-    preset-k8s-ssh: "true"
-  spec:
-    containers:
-    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200623-2424179-1.15
-      args:
-      - --root=/go/src
-      - --repo=k8s.io/kubernetes=release-1.15
-      - --repo=github.com/containerd/cri=release/1.2
```

Comment:
Does cri 1.2 not work with later releases of kubernetes?

Reply:
It might make sense to keep if there is somebody curating it.  Perhaps it is a naive sign of me seeing a test of this odd combination (which mostly passes) and then jumping to an assumption of non-value.  There might be value in a test called "containerd-node-conformance" if that is testing a meaningful variation.  I don't understand what is this "1.2" thing though as there's no such tag on containerd/cri although it is a branch, which hasn't seen commits since January.   There's a 1.3 branch active to March.  And a 1.11 branch active through last year.  Guess this isn't semver.

I would propose these old variations be dropped and an issue opened for a test plan of containerd variations.  Based on `curl http://storage.googleapis.com/k8s-metrics/failures-latest.json | grep -A1 containerd | grep -v ^--`, it seems there's a gap in curation on things containerd:
```
  "ci-cri-containerd-e2e-gce-stackdriver": {
    "failing_days": 889
  "ci-cri-containerd-e2e-gci-gce-sd-logging": {
    "failing_days": 856
  "ci-cri-containerd-e2e-gci-gce-sd-logging-k8s-resources": {
    "failing_days": 847
  "ci-cri-containerd-e2e-gci-gce-statefulset": {
    "failing_days": 808
  "ci-cri-containerd-node-e2e-serial": {
    "failing_days": 760
  "ci-cri-containerd-e2e-gci-gce-es-logging": {
    "failing_days": 651
  "ci-cri-containerd-node-e2e-flaky": {
    "failing_days": 426
  "ci-kubernetes-e2e-windows-containerd-gce": {
    "failing_days": 235
  "ci-kubernetes-e2e-containerd-gce-netd-calico": {
    "failing_days": 232
  "ci-kubernetes-e2e-containerd-gce-netd": {
    "failing_days": 232
  "ci-cri-containerd-e2e-ubuntu-gce": {
    "failing_days": 177
  "ci-containerd-e2e-ubuntu-gce": {
    "failing_days": 177
  "ci-kubernetes-e2e-windows-containerd-gce-1-18": {
    "failing_days": 164
  "ci-cri-containerd-e2e-gci-gce-flaky": {
    "failing_days": 153
  "ci-cri-containerd-e2e-gci-gce-alpha-features": {
    "failing_days": 126
  "ci-containerd-soak-gci-gce": {
    "failing_days": 125
  "pr:pull-cri-containerd-windows-cri": {
    "failing_days": 119
  "ci-cri-containerd-node-e2e-benchmark": {
    "failing_days": 119
  "ci-cri-containerd-cri-validation-windows": {
    "failing_days": 119
  "ci-containerd-node-e2e-features-1-2": {
    "failing_days": 119
  "ci-kubernetes-e2e-aks-engine-azure-master-windows-containerd-hyperv": {
    "failing_days": 87
  "ci-kubernetes-e2e-aks-engine-azure-master-windows-containerd-hyperv-serial-slow": {
    "failing_days": 87
  "ci-containerd-node-e2e-features-1-3": {
    "failing_days": 60
  "ci-containerd-node-e2e-1-3": {
    "failing_days": 60
  "ci-containerd-node-e2e-1-2": {
    "failing_days": 60
  "ci-kubernetes-e2e-gci-gce-scalability-containerd-correctness": {
    "failing_days": 44
  "ci-kubernetes-e2e-aks-engine-master-windows-containerd-azuredisk-csi-driver": {
    "failing_days": 31
```

Despite that some of the boards at https://testgrid.k8s.io/sig-node-containerd do have a lot of green.  It's not clear much of that green is relevant...

Reply:
SGTM can you open an issue?
FYI @vpickard @karan @bsdnet

Reply:
aaand you already did https://github.com/kubernetes/test-infra/issues/18570 thanks